selfdrive/car/subaru/carcontroller.py

```diff
@@ -11,11 +11,9 @@ def __init__(self, car_fingerprint):
     self.STEER_STEP = 2                # how often we update the steer cmd
     self.STEER_DELTA_UP = 50           # torque increase per refresh, 0.8s to max
     self.STEER_DELTA_DOWN = 70         # torque decrease per refresh
-    if car_fingerprint == CAR.IMPREZA:
-      self.STEER_DRIVER_ALLOWANCE = 60   # allowed driver torque before start limiting
-      self.STEER_DRIVER_MULTIPLIER = 10   # weight driver torque heavily
-      self.STEER_DRIVER_FACTOR = 1     # from dbc
-
+    self.STEER_DRIVER_FACTOR = 1       # from dbc
+    self.STEER_DRIVER_ALLOWANCE = 60   # allowed driver torque before start limiting
```

Comment:
we probably need to distinguish between Impreza and others and pass this param to panda safety.